docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -51,10 +51,11 @@ Clerk offers two kinds of authentication strategies: **password** and **password
 
 Choosing the **password** strategy requires users to set a password during the sign up process. Clerk offers out of the box protection against weak and leaked passwords and the only requirement enforced is that the password be a miminum of 8 characters. (**Note**: passwordless authentication is still available to users even if password strategy is selected.)
 
-There are two types of **passwordless** authentication strategies:
+**Passwordless** authentication strategies include:
 
-- **One-time password (OTP)**
-- **Email link**
+- **[Passkeys](/docs/custom-flows/passkeys)**
+- **[One-time password (OTP)](/docs/custom-flows/email-sms-otp)**
+- **[Email link](/docs/custom-flows/email-links)**
```

Comment:
So, custom flows docs should strictly be for custom flows - they should not have the overview of the feature. If you look at this page, otp and email links have dedicated sections below this text. And then they also have dedicated guides for setting up their custom flows.

```suggestion
- **[Passkeys](#passkeys)**
- **[One-time password (OTP)](#one-time-password-otp)**
- **[Email link](#email-link)**
```